Specs/Scene/ExpressionSpec.js

```diff
@@ -3016,7 +3116,7 @@ defineSuite([
         expect(shaderExpression).toEqual(expected);
     });
 
-     it('gets shader expression for sin', function() {
+    it('gets shader expression for sin', function() {
         var expression = new Expression('sin(0.0)');
         var shaderExpression = expression.getShaderExpression('', {});
         var expected = 'sin(0.0)';
```

Comment:
Can you also add tests for `distance`, `dot`, and `cross` for the shader expressions? Similar to 

```
it('gets shader expression for min', function() {
    var expression = new Expression('min(3.0,5.0)');
    var shaderExpression = expression.getShaderExpression('', {});
    var expected = 'min(3.0, 5.0)';
    expect(shaderExpression).toEqual(expected);
});
```